app/services/search/view-search-results.service.js

```diff
@@ -39,15 +30,15 @@ const RETURN_REFERENCE_PATTERN = /^[1-9]\d*$/
 async function go(searchQuery, page) {
   const pageNumber = Number(page)
 
-  const licenceSearchResult = await _searchLicences(searchQuery, pageNumber)
+  const licenceSearchResult = await FetchLicenceSearchResultsService.go(searchQuery, pageNumber)
   const licences = licenceSearchResult.results.length !== 0 ? licenceSearchResult.results : null
 
-  const returnLogSearchResult = await _searchReturnLogs(searchQuery, pageNumber)
+  const returnLogSearchResult = await FetchReturnLogSearchResultsService.go(searchQuery, pageNumber)
```

Comment:
Same comment as for licences.

Reply:
Done, including removing the subsequent line - I've pushed the check for empty results into the presenter